src/RestWrite.js

```diff
@@ -95,6 +95,9 @@ RestWrite.prototype.execute = function() {
     .then(() => {
       return this.runBeforeTrigger();
     })
+    .then(() => {
+      return this.deleteEmailRestTokenIfNeeded();
+    })
```

Comment:
Doh, typo.  Fix me, should be Reset token, not Rest

Reply:
Ohh noooo